### ISSUE TITLE
GPS sometimes fails

### ISSUE THREAD

**Issue by zanco** [darksidelemm/RS41HUP#8]
Hi,
Did fly an 4fsk only tracker today. At some moments the position gets not uploaded because zero Lattitude Longitude not sending, which makes sense, however, the tracker does transmit the right altitude in that same string.

> 15:56:15 [INFO]  Habitat - Uploaded sentence: $$PE2BZ-4FSK,110,13:56:11,52.02987,5.24374,2473,19,0,28,2.78*A93B
15:57:17 [ERROR]  Horus UDP - Zero Latitude/Longitude, not sending.
15:57:18 [INFO]  Habitat - Uploaded sentence: $$PE2BZ-4FSK,111,13:57:13,0.00000,0.00000,2465,0,0,27,2.78*E43C
15:58:19 [ERROR]  Horus UDP - Zero Latitude/Longitude, not sending.
15:58:21 [INFO]  Habitat - Uploaded sentence: $$PE2BZ-4FSK,112,13:58:15,0.00000,0.00000,2391,0,0,27,2.80*6B72
15:59:21 [ERROR]  Horus UDP - Zero Latitude/Longitude, not sending.
15:59:22 [INFO]  Habitat - Uploaded sentence: $$PE2BZ-4FSK,113,13:59:18,0.00000,0.00000,2300,0,0,27,2.80*4D85
16:00:24 [ERROR]  Horus UDP - Zero Latitude/Longitude, not sending.
16:00:25 [INFO]  Habitat - Uploaded sentence: $$PE2BZ-4FSK,114,14:00:20,0.00000,0.00000,2250,0,0,27,2.80*A187
16:01:27 [INFO]  Habitat - Uploaded sentence: $$PE2BZ-4FSK,115,14:01:21,52.02592,5.26813,2151,21,9,28,2.80*0E69

From the config.h  : 
//#define UBLOX_POWERSAVE 1

#define DEEP_SLEEP 2
#define DEEP_SLEEP_PIPS 10000

Can you shine a light on these results ?

Thanks

**Comment by darksidelemm** [comment]
Yep, deep sleep mode isn't working correctly. I've encountered the same issues as you and haven't been able to figure out what's going on as yet. Haven't had the time to debug it any further...
